episodes/02-importing-data.md

```diff
@@ -60,6 +60,10 @@ To import the data for the exercise below, follow the instructions in [Setup](ht
 7. From OpenRefine 3.4 onwards there is an option to Trim leading \& trailing whitespace from strings when importing separator-based files. Keeping this checked will ensure that values like `English` and `English `, which differ by a single trailing space, are not treated as different values after the import
 8. Make sure the `Attempt to parse cell text into numbers` box is not checked, so OpenRefine doesn't try to automatically detect numbers because this could cause errors such as confusion between date formats (e.g. DD/MM/YYYY vs MM/DD/YYYY).
 9. The Project Name box in the upper right corner will default to the title of your imported file. Click in the `Project Name` box to give your project a different name, if desired.
+:::::::::::::: instructor
+This is a good moment to review the points from [What Should I Know When Working with OpenRefine?](01-introduction.md#what-should-i-know-when-working-with-openrefine)
+
+::::::::::::::    
```

Comment:
Insert the instructor note into the list (as per @ostephens recommendation, see: https://pandoc.org/MANUAL.html#block-content-in-list-items)

Note: you can use [the "commit suggestion" button](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) to apply this directly in GitHub

```suggestion
   
   :::::::::::::: instructor
   
   This is a good moment to review the points from [What Should I Know When Working with OpenRefine?](01-introduction.md#what-should-i-know-when-working-with-openrefine)
   
   ::::::::::::::    
```